com.github.k4zmu2a.spacecadetpinball.yml

```diff
@@ -0,0 +1,100 @@
+app-id: com.github.k4zmu2a.spacecadetpinball
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: "pinball"
+finish-args:
+  - "--socket=pulseaudio"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--device=dri"
+  - "--env=SDL_SOUNDFONTS=/app/share/soundfonts/default-GM.sf3"
+  #- "--env=SDL_VIDEODRIVER=wayland"
+modules:
+  - name: SDL2
+    config-opts:
+    - '--disable-static'
+    buildsystem: autotools
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL
+        tag: release-2.0.18
+    modules:
+      - name: libdecor
+        config-opts:
+          - -Ddemo=false
+        buildsystem: meson
+        cleanup:
+          - /include
+          - /lib/pkgconfig
+        sources:
+          - type: git
+            url: https://gitlab.gnome.org/jadahl/libdecor/
+            commit: 8e465f1254a84ff419134df47fa6c3f72b6b784f
+
+  - shared-modules/linux-audio/fluidsynth2.json
+
+  - name: SDL2_mixer
+    buildsystem: autotools
+    config-opts:
+    - '--disable-static'
+    cleanup:
+      - /bin
+      - /include
+      - /lib/pkgconfig
+      - /share
+    sources:
+      - type: git
+        url: https://github.com/libsdl-org/SDL_mixer
+        commit: 2d1d969292861558ad3baba15a6ef869d875ec5b
+
+  - name: FluidR3Mono
+    buildsystem: simple
+    build-commands:
+      - install -D -m 644 -t $FLATPAK_DEST/share/soundfonts FluidR3Mono_GM.sf3
+      - ln -s $FLATPAK_DEST/share/soundfonts/{FluidR3Mono_GM,default-GM}.sf3
+    sources:
+      - type: file
+        url: https://github.com/musescore/MuseScore/raw/3.6.2/share/sound/FluidR3Mono_GM.sf3
+        sha256: 2aacd036d7058d40a371846ef2f5dc5f130d648ab3837fe2626591ba49a71254
+
+  - name: Pinball
+    buildsystem: cmake-ninja
+    no-make-install: true
+    config-opts:
+      - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+    post-install:
+      # Create symlink for game assets
+      - cat assets_list | while read line; do ln -vs ${FLATPAK_DEST}/extra/Pinball/$line ${FLATPAK_DEST}/bin/; done
+      - install -Dm755 apply_extra ${FLATPAK_DEST}/bin/apply_extra
+      - install -Dm755 bin/SpaceCadetPinball ${FLATPAK_DEST}/bin/pinball
+      - install -D -m 644 ${FLATPAK_ID}.desktop -t ${FLATPAK_DEST}/share/applications/
+      - install -Dm644 ${FLATPAK_ID}.metainfo.xml -t ${FLATPAK_DEST}/share/metainfo/
+      - install -Dm644 ${FLATPAK_ID}.png -t ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/
+    sources:
+      - type: git
+        url: "https://github.com/k4zmu2a/SpaceCadetPinball.git"
+        commit: 0dc6006937682e84a2b71f4b22ce30452fa1e11a
```

Comment:
Is this a tag or just some commit? Ideally, it should be using specific release and also use https://github.com/flathub/flatpak-external-data-checker/ for automatic upgrades.

Reply:
I used the latest commit because some time has passed since the last release.

Reply:
But I agree that robots should update that